src/main/java/org/folio/oaipmh/dao/impl/SetDaoImpl.java

```diff
@@ -47,8 +52,8 @@ public SetDaoImpl(final PostgresClientFactory postgresClientFactory) {
   @Override
   public Future<FolioSet> getSetById(String id, String tenantId) {
     return getQueryExecutor(tenantId).transaction(txQE -> {
-      Condition condition = SET.ID.eq(UUID.fromString(id));
-      return txQE.findOneRow(dslContext -> dslContext.selectFrom(SET)
+      Condition condition = SET_LB.ID.eq(UUID.fromString(id));
```

Comment:
dumb question... what does the LB stand for?

Reply:
This decision was taken because there were conflicts with naming of dto's. It clashes with Set java framework class name and such things make bring the confusions to code. As well the generated by jooq pojo's were renamed to due the same reason. Another point is that it allows to declare variables without declaring the full class path name like org.folio.rest.jaxrs.model.Set set = new ...

Reply:
OK that makes sense; but what does it stand for?  Would something like OAI_SET make more sense?

Reply:
LB - means l-liquid b-base: liquibase database entities. The same naming is used in SRS and i think this is okay. I don't think that it is needed to be renamed again.

Reply:
ah, ok got it.